eng/tools/BaselineGenerator/BaselineGenerator.csproj

```diff
@@ -5,10 +5,11 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <StartArguments>-o "$(MSBuildThisFileDirectory)../../Baseline.Designer.props"</StartArguments>
     <StartWorkingDirectory>$(MSBuildThisFileDirectory)../../</StartWorkingDirectory>
+    <LangVersion>preview</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NuGet.Packaging" Version="4.8.0" />
+    <PackageReference Include="NuGet.Protocol" Version="4.8.2" />
```

Comment:
@Eilon please let me know if bumping the NuGet.Packaging version up a couple of patches or expanding the dependencies to include NuGet.Protocol is an issue.

Reply:
Not an issue, thanks for checking.

Reply:
(At least, I don't have an issue from a policy perspective. Not sure if there's any other kind of problem.)